incubator/hnc/e2e/tests/e2e_test.go

```diff
@@ -38,3 +43,14 @@ func tryRun(cmdln ...string) error {
 	GinkgoT().Log("Output: ", string(stdout))
 	return err
 }
+
+func cleanUpNses(nses []string) {
```

Comment:
In function names, it's better to use full words: `cleanUpNamespaces` (or since "cleanup" is often considered a single work `cleanupNamespaces`).

Reply:
done